src/actions/bookmarkArticle.js

```diff
@@ -0,0 +1,34 @@
+import {
+  BOOKMARK_ARTICLE_REQUEST,
+  BOOKMARK_ARTICLE_SUCCESS,
+  BOOKMARK_ARTICLE_FAILURE,
+  UNBOOKMARK_ARTICLE_REQUEST,
+  UNBOOKMARK_ARTICLE_SUCCESS,
+  UNBOOKMARK_ARTICLE_FAILURE,
+} from '../actionTypes/bookmarkArticleActionTypes';
+
+export const bookmarkArticleAction = slug => ({
+  type: BOOKMARK_ARTICLE_REQUEST,
```

Comment:
I thought we agreed to have consistent actions? `type` and `payload `?

Reply:
Yeah. sorry about that. That has been implemented.